### PR TITLE
Highlight current user row in scoreboard

### LINKED ISSUE #86: Highlight current user in scoreboard
TTSIA

### PR DESCRIPTION
**Pull request by mamat-rahmat**
Fixes #86.

### CODE REVIEW

File: application/controllers/contestant/Scoreboard.php [hunk 1]
```diff
@@ -41,6 +42,7 @@ public function index()
 		$this->ui['content']['show_institution_logo'] = $contest['show_institution_logo'];
 		$this->ui['content']['problems'] = $res['problems'];
 		$this->ui['content']['frozen'] = (time() > strtotime($contest['freeze_time'])) && (time() <= strtotime($contest['unfreeze_time']));
+		$this->ui['content']['username'] = $this->session->userdata('username_' . $this->setting->get('sess_id'));
```
Comment (fushar) [inline application/controllers/contestant/Scoreboard.php]: If I remember correctly, you should be able to get the current username as `$this->identity->get_user_name()`

Comment (mamat-rahmat) [reply application/controllers/contestant/Scoreboard.php]: `$this->identity->get_user_name()` retrieves the current user 'name', not the current username of the user. Should i add more method?

Comment (fushar) [reply application/controllers/contestant/Scoreboard.php]: Oh I see. Yes, please

Comment (mamat-rahmat) [reply application/controllers/contestant/Scoreboard.php]: Done

**Pull request merged**